plugin-packs/postcss-preset-env/test/basic.ch38.expect.css

```diff
@@ -368,7 +368,7 @@
 	text-indent: 2em;
 	content: var(--value-2ic);
 	left: 2em;
-	left: var(--non-existing, 2em);
+	left: var(--non-existing, 2ic);
```

Comment:
I can give it the old behavior by moving `custom-properties` to the bottom. But that seemed like a bigger change.

I don't have technical reason not to move `custom-properties`, just felt larger than what I wanted to do now.